docs/installation.md

```diff
@@ -0,0 +1,22 @@
+Open the QGIS plugin manager
+
+## From QGIS repository
+
+Once in the plugin manager, search for STAC APIs. QGIS will provide a list of plugins that have a similar name. If you click on the plugin, you will get its information. Select it, and **Install Plugin**.
+
+![image](images/install-from-repository.png)
+
+## From ZIP
+
+From the Install from ZIP page, select the zip file and click the Install button to install plugin
```

Comment:
we need to provide details about where user can get the zip file for the plugin.